modules/monitoring-common-terms.adoc

```diff
@@ -33,7 +33,11 @@ etcd::
 etcd is the key-value store for {product-title}, which stores the state of all resource objects.
 
 Fluentd::
-Fluentd gathers logs from nodes and feeds them to Elasticsearch.
+Fluentd is a log collector that resides on each [product-title] node. It gathers application, infrastructure, and audit logs and forwards them to different outputs.
```

Comment:
This doesn't replace properly - must be in curly braces. You can check the preview to see.

```suggestion
Fluentd is a log collector that resides on each {product-title} node. It gathers application, infrastructure, and audit logs and forwards them to different outputs.
```

Reply:
Thank you very much for noticing! I have no idea how I missed it in the preview :smile: